plugins/usbdmx/EuroliteProFactory.cpp

```diff
@@ -88,12 +94,18 @@ bool EuroliteProFactory::DeviceAdded(
   // Eurolite USB-DMX512-PRO MK2?
   } else if (descriptor.idVendor == VENDOR_ID_MK2 &&
              descriptor.idProduct == PRODUCT_ID_MK2) {
-    if (m_enable_eurolite_mk2) {
-      OLA_INFO << "Found a possible new Eurolite USB-DMX512-PRO MK2 device";
-      LibUsbAdaptor::DeviceInformation info;
-      if (!m_adaptor->GetDeviceInfo(usb_device, descriptor, &info)) {
-        return false;
-      }
+    LibUsbAdaptor::DeviceInformation info;
+    if (!m_adaptor->GetDeviceInfo(usb_device, descriptor, &info)) {
+      return false;
+    }
+
+    const bool serial_matches = std::find(m_expected_eurolite_serials.begin(),
+      m_expected_eurolite_serials.end(), info.serial) !=
+      m_expected_eurolite_serials.end();
+
+    if (m_enable_eurolite_mk2 || serial_matches) {
```

Comment:
You can just use our STLContains function here:
https://docs.openlighting.org/ola/doc/latest/group__stl.html#ga6bc94dfb62cab831bfa2f0fc7e4daa04

Reply:
Thanks for pointing that out.